packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/form/ViewFormDescriptionConverterSwitch.java

```diff
@@ -443,6 +443,7 @@ public Optional<AbstractControlDescription> caseBarChartDescription(org.eclipse.
         BarChartDescription.Builder builder = BarChartDescription.newBarChartDescription(this.getDescriptionId(viewBarChartDescription))
                 .label(viewBarChartDescription.getName())
                 .labelProvider(this.getStringValueProvider(labelExpression))
+                .targetObjectIdProvider(variableManager -> variableManager.get(IEditingContext.EDITING_CONTEXT, IEditingContext.class).map(IEditingContext::getId).orElse(null))
```

Comment:
Why not self? If a barchart is created here, it is under a group with a self.

Reply:
I would even say that the targetObjectIdProvider here should be the same as the other widget since they all share a common one `.targetObjectIdProvider(this.semanticTargetIdProvider)`

Reply:
I'll update both bar chart and pie chart here with that to merge the PR